docs/platform/server-v3/mailer.md

```diff
@@ -0,0 +1,80 @@
+---
+id: mailer
+title: Mail Component
+sidebar_label: Mail Component
+---
+
+## Overview 
+
+Out of the box our middleware allows emails to be sent easily using our `@deity/falcon-mailer` package. Under the hood this uses [Node Mailer](https://nodemailer.com/about/).
+
+## Configuration
+
+:::note Each mail service is different
+Depending on the service you're using your **smtp** config might vary. Please refer to the [nodemailer docs](https://nodemailer.com/smtp/). We pass the config object as is.
+:::
+
+To configure our mailer component add it to your `server` config as a component. This config might vary depending on your setup.
+
+**`server/config/default.json`**
+```
+"components": {
+    "mailer": {
+      "package": "@deity/falcon-mailer",
+      "config": {
+        "transport": {
+          "service": "[MAILER_SERVICE]",
+          "auth": {
+            "user": "[MAILER_USER]",
+            "pass": "[MAILER_PASSWORD]"
+          },
+        },
+        "send": {
+          "from": "[MAILER_FROM_ADDRESS]"
+        }
+      }
+    }
+  }
+}
+```
+
+- `MAILER_SERVICE` - this is the service you'd like to use. See a list of [supported values here](https://nodemailer.com/smtp/well-known/).
+- `MAILER_USER` - the username for the service above
+- `MAILER_PASSWORD` - the password for the service above
+- `MAILER_FROM_ADDRESS` - the from address, e.g `Falcon Shop <deity.falcon.mailer@gmail.io>`
+
+## Usage
+
+Out of the box we use the mailer packages `sendEmail` method to handle password reset requests.
```

Comment:
@hobadams Probably this sentence I  suggested could be improved but it might be wise to communicate the integrations where `sendEmail` is actually used for this password reset?

```suggestion
Out of the box we use the mailer packages `sendEmail` method to handle password reset requests when the integrated shop doesn't offer this functionality out of the box.
For now this only includes **BigCommerce** and **commercetools** integrations.
```